library/network/test/susefirewalld_services_test.rb

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env rspec
+
+require_relative "test_helper"
+
+Yast.import "SuSEFirewallServices"
+Yast.import "SCR"
+Yast.import "FileUtils"
+
+describe Yast::SuSEFirewallServices do
+
+  before :example do
+    # We shouldn't run this test if firewalld is running
+    if Yast::SuSEFirewallServices.is_a?(Yast::SuSEFirewall2ServicesClass)
+      skip "SuSEfirewall2 backend is not supported by this test" do
```

Comment:
Yes but I do not know how to fix that properly to be honest... It's the problem I mentioned in my original request that tests run in parallel and we can't have both backends running.

As for the failures, I can't reproduce it here on tubleweed. I will try again but if you could tell me what exactly is failure that would be great. FWIW
```
tw1:/yast-yast2 # systemctl is-active firewalld 
active

PASS: systemctl_test.rb
PASS: systemd_unit_test.rb
PASS: systemd_socket_test.rb
PASS: systemd_target_test.rb
PASS: systemd_service_test.rb
============================================================================
Testsuite summary for yast2 3.1.183
============================================================================
# TOTAL: 5
# PASS:  5
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
```